_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -280,4 +401,62 @@ Links created via scripting, but without the semantic role of link:
 </span>
 ```
 
+#### Inapplicable Example 5
+
+These links have empty [accessible names][accessible name].
+
+```html
+<a href="http://facebook.com"></a> <a href="http://twitter.com"></a>
+```
+
+#### Inapplicable Example 6
+
+These image links have empty [accessible names][accessible name].
+
+```html
+<a href="http://facebook.com"><img src="facebook.jpg"/></a> <a href="http://twitter.com"><img src="twitter.jpg"/></a>
+```
+
+#### Inapplicable Example 7
+
+Only the first link ("all the time") is present in the [flat tree][]. Hence, there is no set of two or more links to apply the rule. The [shadow tree][] does contain a [slot][], but because its [named][name], the second link ("from the light") is _not_ [slotted][slot] into it.
```

Comment:
This is more a test for shadow DOM than it is for links. IMO this should be removed.